homeassistant/components/dreo/config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Config flow to configure Dreo."""
+
+import hashlib
+import logging
+from typing import Any
+
+from hscloud.hscloud import HsCloud
+from hscloud.hscloudexception import HsCloudBusinessException, HsCloudException
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+class DreoFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Handle a Dreo config flow."""
+
+    VERSION = 1  # 添加版本号
```

Comment:
```suggestion
    VERSION = 1
```

Comment should be in English so everybody can understand it